### PR TITLE
Problem: missing account and account message projection unit test

### PR DESCRIPTION
**Pull request by davcrypto**
Solution: fixed ( #543 and #544 )

### CODE REVIEW

File: projection/account/account_test.go [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+		TypeConv:    &pg.PgxTypeConv{},
+		StmtBuilder: pg.PostgresStmtBuilder,
+	}).Maybe()
+	mockTx.On("Rollback").Return(nil).Maybe()
+	mockTx.On("Commit").Return(nil).Maybe()
+
+	return mockTx
+}
+
+func TestIBCChannel_HandleEvents(t *testing.T) {
```
Comment (ysong42) [inline projection/account/account_test.go:53]:
```suggestion
func TestAccount_HandleEvents(t *testing.T) {
```

File: projection/account_message/account_message_test.go [hunk 2]
```diff
@@ -0,0 +1,4264 @@
...
+		TypeConv:    &pg.PgxTypeConv{},
+		StmtBuilder: pg.PostgresStmtBuilder,
+	}).Maybe()
+	mockTx.On("Rollback").Return(nil).Maybe()
+	mockTx.On("Commit").Return(nil).Maybe()
+
+	return mockTx
+}
+
+func TestIBCChannel_HandleEvents(t *testing.T) {
```
Comment (ysong42) [inline projection/account_message/account_message_test.go:57]:
```suggestion
func TestAccountMessage_HandleEvents(t *testing.T) {
```

**Review by ysong42 — COMMENTED** [review]
Found some easy to forget parts when doing copy and paste 😂 
I also found this in my PR

**Comment by davcrypto** [comment]
@ysong42 Thanks for reviewing. Updated those issues.

**Review by calvinaco — APPROVED** [review]
LGTM!

**Pull request merged**